lib/class-wp-rest-plugins-controller.php

```diff
@@ -43,13 +43,22 @@ public function register_routes() {
 	 * @return WP_Error|boolean
 	 */
 	public function get_items_permissions_check( $request ) {
-
-		if ( ! current_user_can( 'manage_options' ) ) { // TODO: Something related to plugins. activate_plugin capability seems to not be available for multi-site superadmin (?)
-			return new WP_Error( 'rest_forbidden', __( 'Sorry, you cannot view the list of plugins' ), array( 'status' => rest_authorization_required_code() ) );
+		if ( is_multisite() ) {
+			$menu_permissions = get_site_option( 'menu_items' );
```

Comment:
On a multisite install, regular site admins (non super admins),  can view and manage plugins specific to their site, when this option is enabled.  I wanted to reflect that core capability in the permissions check here.  I will work on making the conditionals more clear.  If it is not apparent then it is bad code.  I will try to make the intent more clear.